### ISSUE TITLE
`aria-hidden` does not have explicit 'true' value resulting in being ignored by accessibility tools

### ISSUE THREAD

**Issue by maartenottenhof** [chakra-ui/zag#2248]
# 🐛 Bug report

`aria-hidden` does not have an explicit 'true' value resulting in being ignored by accessibility tools. Whenever you interact with a component which should result in setting `aria-hidden` to other elements on the page, this does not have any effect. This seems to happen because of the empty value of `aria-hidden`. Regarding the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden), `aria-hidden` should have an explicit true value like `aria-hidden="true"` instead of just adding an empty attribute like `aria-hidden=""`.

## 💥 Steps to reproduce

1. Click on 'Open Dialog'
2. Open the browsers accessibility checker
3. See elements that should be hidden by `aria-hidden` as accessible

## 💻 Link to reproduction

CodeSandbox reproduction: https://stackblitz.com/edit/zamo1mds?file=src%2FDialog.tsx

## 🧐 Expected behavior

`aria-hidden="true"` instead of `aria-hidden=""` so elements are properly hidden.

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      |     0.82.1       |
| Browser          |     Chrome 133.0.6943.127        |
| Operating System |     macOS 15.3.1 (24D70)       |

## 📝 Additional information

<img width="425" alt="Image" src="https://github.com/user-attachments/assets/acbb8e0e-cf96-46ca-a331-846fa6f13cc8" />

<img width="447" alt="Image" src="https://github.com/user-attachments/assets/5d602046-31bd-4c75-b7bf-8e1063d60a6a" />
